scripts/create_gitlab_snapshot.py

```diff
@@ -0,0 +1,107 @@
+    instance = gitlab_instance_info()
+    shutdown_instance(instance)
+    create_snapshot(instance)
+
+
+def gitlab_instance_info() -> Mapping[str, JSON]:
+    filters_ = [
+        {
+            'Name': 'key-name',
+            'Values': ['azul-gitlab']
+        }
+    ]
+    return one([
+        one(reservation['Instances'])
+        for reservation in aws.ec2.describe_instances(Filters=filters_)['Reservations']
+    ])
+
+
+def shutdown_instance(instance: Mapping[str, JSON]):
+    instance_ids = [instance['InstanceId']]
+    log.info('Preparing to stop GitLab instance for %s, waiting 10 seconds before proceeding',
+             config.deployment_stage)
+    sleep(10)
+    log.info('Stopping instance %s', instance_ids)
+    aws.ec2.stop_instances(InstanceIds=instance_ids)
+    try:
+        log.info('Waiting up to 10 minutes for the instance to stop')
+        waiter = aws.ec2.get_waiter('instance_stopped')
+        waiter.wait(InstanceIds=instance_ids)
+    except botocore.exceptions.WaiterError as e:
+        log.warning(e)
+    else:
+        log.info('Instance %s has stopped', instance_ids)
+
+
+def create_snapshot(instance: Mapping[str, JSON]):
+    # FIXME: Consolidate uses of sts.get_caller_identity
+    #        https://github.com/DataBiosphere/azul/issues/3890
+    role_id = aws.sts.get_caller_identity()['UserId'].split(':')[0]
+    resource_tags = [
+        {'Key': 'component', 'Value': 'azul-gitlab'},
+        {'Key': 'created-by', 'Value': role_id},
+        {'Key': 'deployment', 'Value': config.deployment_stage},
+        {'Key': 'Name', 'Value': 'azul_gitlab'},
+        {'Key': 'owner', 'Value': config.owner},
+        {'Key': 'project', 'Value': 'dcp'},
+        {'Key': 'service', 'Value': 'azul'},
+    ]
+    date = datetime.datetime.now().strftime('%Y-%m-%d %H:%M')
+    response = aws.ec2.create_snapshots(Description=f'{date} Gitlab Update',
+                                        InstanceSpecification=dict(InstanceId=instance['InstanceId'],
+                                                                   ExcludeBootVolume=True),
+                                        TagSpecifications=[dict(Tags=resource_tags,
+                                                                ResourceType='snapshot')])
+    snapshot = [s['SnapshotId'] for s in response['Snapshots']]
+    log.info('Created snapshot %s for %s, waiting up to 30 minutes for completion',
+             snapshot, instance['KeyName'])
+    try:
+        waiter = aws.ec2.get_waiter('snapshot_completed')
+        waiter.wait(SnapshotIds=snapshot,
+                    WaiterConfig={
+                        'MaxAttempts': 120,
+                        'Delay': 15
+                    })
+    except botocore.exceptions.WaiterError as e:
+        log.warning(e)
+        log.warning('Verify the snapshot status in the AWS Console')
+    else:
+        log.info('Snapshots for %s in %s are complete',
```

Comment:
```suggestion
        log.info('Snapshot for %s in %s are complete',
```

Please make sure you fix this everywhere this time.